content/docs/quickstart/cpp.md

```diff
@@ -9,63 +9,111 @@ description: This guide gets you started with gRPC in C++ with a simple working
 
 #### gRPC
 
-To install gRPC on your system, follow the [instructions to install gRPC C++ via make](https://github.com/grpc/grpc/blob/master/src/cpp/README.md#make).
+Follow the gRPC installation instructions for your OS:
 
-To run the example code, please ensure `pkg-config` is installed on your
-machine before you build and install gRPC in the previous step, since the
-example `Makefile`s try to look up the installed gRPC path using `pkg-config`.
-On Debian-based systems like Ubuntu, this can usually be done via
-`sudo apt-get install pkg-config`.
+- macOS:
```

Comment:
nit: It feels a bit odd to include macOS as a special case.

Reply:
Agreed, but I wouldn't want to omit the brew-based install for macOS users merely because it is the _only_ platform that supports installation other than from sources.

Are you aware of a similar way to install binaries under Linux and/or Windows?

Reply:
Remove